src/v2/Apps/Consign/Routes/MarketingLanding/Components/PromoSpace.tsx

```diff
@@ -1,24 +1,22 @@
 import React from "react"
-import { Link, Text } from "@artsy/palette"
+import { Text, Box, color } from "@artsy/palette"
 import { SectionContainer } from "./SectionContainer"
-import { Media } from "v2/Utils/Responsive"
 
 export const PromoSpace: React.FC = () => {
   return (
-    <SectionContainer bg="black5" py={3}>
-      <Media greaterThanOrEqual="sm">
-        <Text variant="subtitle" color="black100" textAlign="center">
-          Gallerist or art dealer?{" "}
-          <Link href="https://partners.artsy.net">Become a partner</Link> to
-          access the largest global online art marketplace
+    <SectionContainer>
+      <Box borderBottom={`1px solid ${color("black60")}`} width="100%">
```

Comment:
`borderBottom="1px solid" borderColor="black60"` — try to avoid the `color` and `space` helpers — they aren't ever actually necessary and I'll maybe actually get around to deprecating them in Palette today 😅